typescript/sdk/src/token/EvmERC20WarpRouteReader.ts

```diff
@@ -142,9 +145,18 @@ export class EvmERC20WarpRouteReader extends EvmRouterReader {
       : undefined;
     const destinationGas = await this.fetchDestinationGas(warpRouteAddress);
 
+    const hasRebalancingInterface =
+      compareVersions(
+        tokenConfig.contractVersion!,
+        REBALANCING_CONTRACT_VERSION,
+      ) > 0;
```

Comment:
Shouldn't this be `>= 0` because if the version matches we also want to fetch these values no? Checking the docs 0 is returned if the versions are equal which means we can fetch the new fields (probably the reason why tests are failing too) 
ref: https://www.npmjs.com/package/compare-versions